### PR TITLE
 indy_vdr specified version because it occurs when installing as pip3…

### PR DESCRIPTION
**Pull request by mingi3442**
An error occurs while installing indy_vdr when running a demo.
Verified normal operation if the version is specified.
This request specifies the version of indy_vdr

### CODE REVIEW

**Comment by swcurran** [comment]
Thanks for the PR!

@usingtechnology @andrewwhitehead — I assume this is needed for now. We’ll have to update this again when the final Indy VDR 0.4.0 goes out, right?  Any other options?

Thanks

**Comment by usingtechnology** [comment]
right now, indy-vdr is at `dev16`, so gets out of date quickly. not sure if we want indy-vdr to be published with `latest`. That still won't help when 0.4.0 is final and published.  We will have to return to this and update requirements to the final 0.4.0 version.

**Comment by swcurran** [comment]
OK — so let’s go ahead and merge this, and then we’ll track the need to update it when indy-vdr is updated.

File: requirements.txt [hunk 1]
```diff
@@ -5,4 +5,6 @@ base58
 multidict<5.0.0
 yarl==1.6.0
 
-indy_vdr
\ No newline at end of file
+# indy_vdr
+# Error loading library: indy_vdr specified version because it occurs when installing as pip3 install indy-vdr.
+indy_vdr==0.4.0.dev1
```
Comment (swcurran) [inline requirements.txt:10]: Please update to “0.4.0.dev16” and confirm it is still working.  Sorry I missed that dev1 was not the latest.

Thanks

Comment (mingi3442) [reply requirements.txt]: After modifying to dev16, it was confirmed that it also works normally when running.
And then I commit again

**Review by swcurran — APPROVED** [review]

**Comment by swcurran** [comment]
Thanks!

**Pull request merged**